sensor/admission-control/manager/images.go

```diff
@@ -80,21 +80,22 @@ func (m *manager) getImageFromSensorOrCentral(ctx context.Context, s *state, img
 	resp, err := m.client.GetImage(ctx, &sensor.GetImageRequest{
 		Image:      img,
 		ScanInline: s.GetClusterConfig().GetAdmissionControllerConfig().GetScanInline(),
+		Namespace:  deployment.GetNamespace(),
 	})
 	if err != nil {
 		return nil, err
 	}
 	return resp.GetImage(), nil
 }
 
-func (m *manager) fetchImage(ctx context.Context, s *state, resultChan chan<- fetchImageResult, pendingCount *int32, idx int, image *storage.ContainerImage) {
+func (m *manager) fetchImage(ctx context.Context, s *state, resultChan chan<- fetchImageResult, pendingCount *int32, idx int, image *storage.ContainerImage, deployment *storage.Deployment) {
 	defer func() {
 		if atomic.AddInt32(pendingCount, -1) == 0 {
 			close(resultChan)
 		}
 	}()
 
-	scannedImg, err := m.getImageFromSensorOrCentral(ctx, s, image)
+	scannedImg, err := m.getImageFromSensorOrCentral(ctx, s, image, deployment)
```

Comment:
Post MVP will also be sending the list of pull secret names and possibly the service account name - I felt it was cleaner to pass the pointer to the deployment instead of passing the individual fields. I don't mind either way, if you have a strong opinion on it I will change it to the individual fields post MVP